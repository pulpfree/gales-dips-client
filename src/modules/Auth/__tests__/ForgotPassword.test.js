/* eslint no-undef: "off" */

import React from 'react'
import { shallow } from 'enzyme'

import ForgotPassword from '../ForgotPassword'

describe('ForgotPassword', () => {
  let mountedC
  let props
  const reportC = () => {
    if (!mountedC) {
      mountedC = shallow(<ForgotPassword {...props} />)
    }
    return mountedC
  }

  beforeEach(() => {
    props = {
      authState: 'forgotPassword',
      onStateChange: () => {},
    }
    mountedC = undefined
  })

  it('matches snapshot', () => {
    expect(reportC()).toMatchSnapshot()
  })
})
