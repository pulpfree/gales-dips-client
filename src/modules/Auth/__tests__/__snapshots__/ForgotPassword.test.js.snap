// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPassword matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(ForgotPassword)
    authState="forgotPassword"
    onStateChange={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "authState": "forgotPassword",
      "classes": Object {
        "container": "ForgotPassword-container-1",
        "formContainer": "ForgotPassword-formContainer-2",
        "formControl": "ForgotPassword-formControl-3",
        "returnButton": "ForgotPassword-returnButton-4",
        "submitButton": "ForgotPassword-submitButton-5",
      },
      "onStateChange": [Function],
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "authState": "forgotPassword",
        "classes": Object {
          "container": "ForgotPassword-container-1",
          "formContainer": "ForgotPassword-formContainer-2",
          "formControl": "ForgotPassword-formControl-3",
          "returnButton": "ForgotPassword-returnButton-4",
          "submitButton": "ForgotPassword-submitButton-5",
        },
        "onStateChange": [Function],
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
